### ISSUE TITLE
Is there a use case for several models per net.yaml?

### ISSUE THREAD

**Issue by m-philipps** [sebapersson/petab_sciml#7]
At the moment in the test cases and [standard](https://github.com/sebapersson/petab_sciml/blob/a5d2ffdd2067c318919943475f9cf97c2bfeceba/src/python/petab_sciml/standard/schema.yaml#L107-L111), the net.yaml has an array of `models`. 
Are multiple models required, considering that the test cases always use different yaml files for several nets?

**Comment by dilpath** [comment]
Not required IMO, and probably nicer to switch to one-model-per-YAML in the sciml spec.

**Comment by sebapersson** [comment]
I agree that we should just have one model per YAML, so I think we can change in mkstd to have 1 model per yaml then.
